src/generator/generator.ts

```diff
@@ -211,6 +220,29 @@ export class Generator {
     await this.render('root-index.njk', {apis}, rootIndexPath);
   }
 
+  async getPkgPathAndData(
+    apisPath: string,
+    file: string,
+    desc: string,
+    defaultVersion = '0.1.0'
+  ): Promise<[string, PkgData]> {
+    const pkgPath = path.join(apisPath, file, 'package.json');
+    const packageData = {name: file, desc, version: defaultVersion};
+    // Use the version from the existing package.json, if possible:
+    try {
+      const pkgRaw = await readFile(pkgPath, 'utf8');
+      const pkg = JSON.parse(pkgRaw);
+      packageData.version = pkg.version;
+    } catch (err) {
+      if (err.code === 'ENOENT') {
+        console.info(`${pkgPath} not found`);
```

Comment:
Not sure if we need the output here.  This will just happen for any case where there's a new API being shipped.  Should we drop it?

Reply:
@JustinBeckwith I thought it might help us debug if we're finding it's not working, I don't feel strongly if you'd rather cut down on noise.

Reply:
I don't feel strongly either (I put an approve on it)